pythonparser/ast.py

```diff
@@ -57,6 +57,15 @@ def value(name):
                            self._reprfields()))
         return "%s(%s)" % (self.__class__.__name__, fields)
 
+    # Compat with stdlib
+    @property
+    def lineno(self):
+        return self.loc
```

Comment:
You might have meant `self.loc.line()`? Then it would also make sense to add a `def col_offset()`. On the other hand I don't see any point in setters, since trying to fixup every loc field is a waste of time.